libs/auth-api-lib/seeders/data/client-licenses-rls.ts

```diff
@@ -3,9 +3,15 @@ import { compose, createClient, createScope } from './helpers'
 export const up = compose(
   createScope({
     name: '@island.is/licenses:firearm',
-    displayName: 'RLS digital license update scope',
+    displayName: 'RLS digital firearm license update scope',
     description:
-      'Veitir aðgang að uppfærsluvirkni á skírteinum útgefnum af RLS',
+      'Veitir aðgang að uppfærsluvirkni á skotvopnaskírteinum útgefnum af RLS',
```

Comment:
As this scope already exists this unfortunately won't update the scope display name and description. You can request the change in the #ids_changes slack channel.